src/admin/collectionsOrBundles/helpers/collection-filters.ts

```diff
@@ -184,6 +184,13 @@ export function generateCollectionWhereObject(
 		},
 	});
 
+	const excludeChannelType = get(filters, 'excludeChannelType');
```

Comment:
You can use the native option for these from now on:
```suggestion
	const excludeChannelType = filters?.excludeChannelType;
```

It provides better type checking.